MineSweeper.py

```diff
@@ -0,0 +1,256 @@
+import Tkinter as tk
+import random
+
+import sys
+sys.path.append('~/Library/Python/2.7/site-packages')
+
+from drawingpanel import *
+
+BOARD_HEIGHT = 300
+BOARD_WIDTH = 300
+BOX_SIZE = 20
+
+NUMBER_MINES = ((BOARD_WIDTH / BOX_SIZE - 1)**2)//10 
+
+class Game():
+
+  def __init__(self):
+    self.master = tk.Tk()
+    self.panel = Canvas(self.master, width = BOARD_WIDTH, height = BOARD_HEIGHT + 100, bg = 'white')
+    self.panel.pack()
+
+
+     
+  def game_over(self):
+
+    self.master.unbind("<Button-1>")
+    self.master.unbind("<Button-2>")
+
+    self.b = Button(self.master, text = 'Play Again?', command = self.new_game)
+    self.panel.create_text(BOARD_WIDTH/2, BOARD_HEIGHT + 40, text  = 'Game Over', fill = 'red') 
+    self.b.pack()
+
+  def new_game(self):
+
+    self.master.destroy()
+
+    main()
```

Comment:
Calling `main` from here is *really* confusing. Try to figure a way to not have to do that.